Cargo.toml

```diff
@@ -20,7 +20,7 @@ description = "Neqo, the Mozilla implementation of QUIC in Rust."
 keywords = ["quic", "http3", "neqo", "mozilla", "ietf", "firefox"]
 categories = ["network-programming", "web-programming"]
 readme = "README.md"
-version = "0.12.0"
+version = "0.12.1"
```

Comment:
Note that this pull request bumps the patch version to `v0.12.1`.

That will allow me to cut a new release once merged in order to downstream it into mozilla-central.

Let me know if you prefer a separate pull request.

Reply:
Fine by me. Do you also want to bump `mtu` now or later?

Reply:
For the record, see https://github.com/mozilla/neqo/pull/2379.